pkg/services/ngalert/notifier/channels/discord.go

```diff
@@ -96,69 +134,78 @@ func newDiscordNotifier(fc channels.FactoryConfig) (*DiscordNotifier, error) {
 func (d DiscordNotifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
 	alerts := types.Alerts(as...)
 
-	bodyJSON := simplejson.New()
+	msg := discordMessage{}
```

Comment:
A small change to make it more idiomatic. We've been using this form everywhere to declare zero-values, including the `tmplErr` below.
```suggestion
	var msg discordMessage
```